.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I have seen other git repos that have added things like `.git` to this file. For example, [docker/compose's .dockerignore](https://github.com/docker/compose/blob/master/.dockerignore)

Reply:
Done

Reply:
I suppose we'll have to duplicate this docker ignore in each repo. lib and build probably don't matter much here.